README.md

```
@@ -543,6 +543,46 @@ send_notification_task.delay(
+
+### Configuration
+
+Add your server-side environment key to `.env`:
+
+```bash
+FLAGSMITH_ENVIRONMENT_KEY=<your_key>
+```
+
+### Usage
+
+**1. Route Dependency**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)